### DISCUSSION TITLE
Remember Upload Custom Remote Before 9th Novemeber....

### DISCUSSION THREAD

**Discussion by highveld66** [GerryDazoo/Slinger#138] [General]
Not sure 
But I suspect you wont be able to upload a custom remote to a Slingbox after 9th Nov when things gt shutdown.
If there is a way I would be interested in knowing
But thought I would raise this as a possible gotchya for post 9th Nov

**Comment by GerryDazoo** [comment]
We're working on it. But yes, JIC configure your slingboxes before the 9th. Do not do a factory reset after the 9th.

**Reply by smt171** [reply]
If we already have a working remote for the normal Sling Box do we need to back it up or anything? Will the remote continue to work after Nov 9, 2022?

**Reply by GerryDazoo** [reply]
Yes, it will work. All the info needed is stored on the slingbox.

**Comment by makaveli3004** [comment]
Good Evening everyone,  I just want to say how thankful I am I was directed to this guide and shout out to Gerry for being so kind and helpful. Now I am new to this and want to get started setting up my sling 500 for Nov 9th, I was able to log in and get my admin password. My question is can this set up be done remotely. I am currently in Florida and my Sling is located in NY. I wont be home for a few months so well beyond Nov 9th.

**Reply by GerryDazoo** [reply]
When you say "My question is can this set up be done remotely". Are you referring to setting up the slinger software of configuring the remotes? If it's the remotes then just give it a try but most people won't need to do this. If it's the slinger setup you are referring to then
you will need to add a port forwarding rule to the router in NY to allow the server to access the slingbox. So if you don't have any kind of remote access you'll have to wait.
